File: seeker/snippet/Calculator.java
//date: 2023-01-13T17:08:59Z
//url: https://api.github.com/gists/3d46061af085121a0b47883e63e0360e
//owner: https://api.github.com/users/akbarsiddique

import java.util.*;
public class Calculator {
public static void main(String[] args){
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter  a :");
    int a= sc.nextInt();
    System.out.println("Enter b :");
    int b=  sc.nextInt();
    System.out.println("Enter opretor :");
    char operator = sc.next().charAt(0);
    switch (operator){
        case '+':
            System.out.println(a+b);
            break;
        case '-':
            System.out.println(a-b);
            break;
        case '*':
            System.out.println(a*b);
            break;
        case '/':
            System.out.println(a/b);
            break;
        case '%':
            System.out.println(a%b);
            break;

    }

}
}

File: seeker/snippet/calculator.py
#date: 2023-05-12T16:52:38Z
#url: https://api.github.com/gists/323c6c2795f563ee55f0d58cb87d4f6f
#owner: https://api.github.com/users/AminRst

enter = input('Enter: ').replace(' ','')
x = 0
y = 0
z = 0
v1 = 0
lst = []
for i in enter:
    if ord(i) < 48:
        lst.append(i)

if len(lst) == 1:
    x = int(enter[:enter.index(lst[0])])
    y = int(enter[(enter.index(lst[0]))+1:])
if len(lst) == 2:
    if lst[0] == '*' or lst[0] == '/':
        x = int(enter[:enter.index(lst[0])])
        y = int(enter[(enter.index(lst[0])) + 1: (enter.rindex(lst[1]))])
        z = int(enter[(enter.rindex(lst[1]))+1:])
    elif lst[1] == '*' or lst[1] == '/' :
        if lst[0] == '+':
            z = int(enter[:enter.index(lst[0])])
            x = int(enter[(enter.index(lst[0])) + 1: (enter.rindex(lst[1]))])
            y = int(enter[(enter.rindex(lst[1])) + 1:])
            lst.reverse()
        elif lst[0] == '-':
            z = -int(enter[:enter.index(lst[0])])
            x = -int(enter[(enter.index(lst[0])) + 1: (enter.rindex(lst[1]))])
            y = int(enter[(enter.rindex(lst[1])) + 1:])
            lst.reverse()
    else:
        x = int(enter[:enter.index(lst[0])])
        y = int(enter[(enter.index(lst[0])) + 1: (enter.rindex(lst[1]))])
        z = int(enter[(enter.rindex(lst[1])) + 1:])

if lst[0] == '+':
    v1 = x + y
    if len(lst) == 1:
        print(v1)
elif lst[0] == '-':
    v1 = x - y
    if len(lst) == 1:
        print(v1)
elif lst[0] == '*':
    v1 = x * y
    if len(lst) == 1:
        print(v1)
elif lst[0] == '/':
    v1 = x / y
    if len(lst) == 1:
        print('%.2f' % v1)

if len(lst) == 2:
    if lst[1] == '+' and v1 != 0:
        print(v1 + z)
    elif lst[1] == '-' and v1 != 0:
        print( v1 - z)
    elif lst[1] == '*' and v1 != 0:
        print(v1 * z)
    elif lst[1] == '/' and v1 != 0:
        print(v1 / z)